src/components/data-management/SamplesTable.jsx

```diff
@@ -275,12 +275,25 @@ const SamplesTable = forwardRef((props, ref) => {
     setTableData(newData);
   }, [projects, samples, activeProjectUuid]);
 
-  const downloadPublicDataset = async () => {
-    const s3Object = await Storage.get('PBMC_3k.zip',
+  const sampleData = [
```

Comment:
this is a list of datasets not samples, change the name

Is there any reason to add all this code to be able to download 2 datasets instead of just changing the default one?

Reply:
Whoops, it should be `exampleDatasets`. I was confusing `sample` with `example`.

Reply:
I talked to Vicky about replacing the default file to use the new example dataset. She thinks that we shouldn't remove access to the original 3k PBMC dataset. I agree with her - there is merit to still have the 3k PBMC dataset. It is smaller and faster to process, compared to the PBMC BMMC dataset. So, if we only need to demo quickly, we can use the 3k PBMC dataset.

Having the list will also allow us to easily add more example datasets in the future, if we want to show case a specific part of the platform (e.g. spatial?)